Project won't  build after adding razorpay-cordova, presumably because of missing frameworks

I'm not able to build my project after adding the plugin. Here's the stacktrace I get when I try to build the project.

```
Ld build/emulator/afitgo.app/afitgo normal i386
    cd /Users/gaurav/Workspace/afitgo/afitgo-app/platforms/ios
    export IPHONEOS_DEPLOYMENT_TARGET=8.0
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -arch i386 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator9.2.sdk -L/Users/gaurav/Workspace/afitgo/afitgo-app/platforms/ios/build/emulator -F/Users/gaurav/Workspace/afitgo/afitgo-app/platforms/ios/build/emulator -Fafitgo/Plugins/com.razorpay.cordova -filelist /Users/gaurav/Workspace/afitgo/afitgo-app/platforms/ios/build/afitgo.build/Debug-iphonesimulator/afitgo.build/Objects-normal/i386/afitgo.LinkFileList -Xlinker -rpath -Xlinker @executable_path/Frameworks -mios-simulator-version-min=8.0 -Xlinker -objc_abi_version -Xlinker 2 -ObjC -fobjc-arc -fobjc-link-runtime /Users/gaurav/Workspace/afitgo/afitgo-app/platforms/ios/build/emulator/libCordova.a -framework Razorpay -Xlinker -dependency_info -Xlinker /Users/gaurav/Workspace/afitgo/afitgo-app/platforms/ios/build/afitgo.build/Debug-iphonesimulator/afitgo.build/Objects-normal/i386/afitgo_dependency_info.dat -o /Users/gaurav/Workspace/afitgo/afitgo-app/platforms/ios/build/emulator/afitgo.app/afitgo
Undefined symbols for architecture i386:
  "_OBJC_CLASS_$_CTTelephonyNetworkInfo", referenced from:
      objc-class-ref in Razorpay(RazorpayLogging.o)
  "_SCNetworkReachabilityCreateWithName", referenced from:
      -[RazorpayLogging isWiFi] in Razorpay(RazorpayLogging.o)
      -[RazorpayLogging isWWAN] in Razorpay(RazorpayLogging.o)
  "_SCNetworkReachabilityGetFlags", referenced from:
      -[RazorpayLogging isWiFi] in Razorpay(RazorpayLogging.o)
      -[RazorpayLogging isWWAN] in Razorpay(RazorpayLogging.o)
ld: symbol(s) not found for architecture i386
clang: error: linker command failed with exit code 1 (use -v to see invocation)

** BUILD FAILED **


The following build commands failed:
	Ld build/emulator/afitgo.app/afitgo normal i386
(1 failure)
ERROR building one of the platforms: Error code 65 for command: xcodebuild with args: -xcconfig,/Users/gaurav/Workspace/afitgo/afitgo-app/platforms/ios/cordova/build-debug.xcconfig,-project,afitgo.xcodeproj,ARCHS=i386,-target,afitgo,-configuration,Debug,-sdk,iphonesimulator,build,VALID_ARCHS=i386,CONFIGURATION_BUILD_DIR=/Users/gaurav/Workspace/afitgo/afitgo-app/platforms/ios/build/emulator,SHARED_PRECOMPS_DIR=/Users/gaurav/Workspace/afitgo/afitgo-app/platforms/ios/build/sharedpch
You may not have the required environment or OS to build this project
Error: Error code 65 for command: xcodebuild with args: -xcconfig,/Users/gaurav/Workspace/afitgo/afitgo-app/platforms/ios/cordova/build-debug.xcconfig,-project,afitgo.xcodeproj,ARCHS=i386,-target,afitgo,-configuration,Debug,-sdk,iphonesimulator,build,VALID_ARCHS=i386,CONFIGURATION_BUILD_DIR=/Users/gaurav/Workspace/afitgo/afitgo-app/platforms/ios/build/emulator,SHARED_PRECOMPS_DIR=/Users/gaurav/Workspace/afitgo/afitgo-app/platforms/ios/build/sharedpch
```

--- a/plugin.xml
+++ b/plugin.xml
@@ -44,5 +44,7 @@
       <header-file src="src/ios/Main.h" target-dir="Razorpay"/>
       <source-file src="src/ios/Main.m" target-dir="Razorpay"/>
       <framework src="src/ios/Razorpay.framework" custom="true" />
+      <framework src="CoreTelephony.framework" />
+      <framework src="SystemConfiguration.framework" />
   </platform>
 </plugin>

